src/status_im/contexts/wallet/create_account/edit_derivation_path/view.cljs

```diff
@@ -1,16 +1,51 @@
 (ns status-im.contexts.wallet.create-account.edit-derivation-path.view
   (:require
     [quo.core :as quo]
+    [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [react-native.safe-area :as safe-area]
     [reagent.core :as reagent]
+    [status-im.constants :as constants]
     [status-im.contexts.wallet.common.temp :as temp]
     [status-im.contexts.wallet.common.utils :as utils]
     [status-im.contexts.wallet.create-account.edit-derivation-path.style :as style]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn path-format-sheet
+  []
+  (let [{:keys [customization-color]} (rf/sub [:get-screen-params])]
```

Comment:
can we adjust this to not get customization-color from screen params. We should be getting this from re-frame, no?

Reply:
@J-Son89 nope, this is the color selected while creating an account. It is not yet in re-frame

Reply:
Hmm, probably we need to move those details to reframe?

Reply:
@J-Son89 why? The account is not yet created, the color is stored in the screen's internal state. Optionally while creating an account the user can edit the derivation path so we path the color along when navigating

Reply:
You mean 'pass the color' along right?
If so mostly to avoid prop drilling like that. It becomes a pain to develop this way if flows are extended or changed.

We can leave for this pr but it's worth considering if we need to refactor the data in the create account pages and have it in reframe instead.

Reply:
@J-Son89 yes I mean 'pass the color along'

> We can leave for this pr but it's worth considering if we need to refactor the data in the create account pages and have it in reframe instead.
Okay we can consider and discuss that with the team